### PR TITLE
Plot segment properties

### PR DESCRIPTION
**Pull request by zBritva**

### CODE REVIEW

**Comment by zBritva** [comment]
@stopyoukid Could you review this PR too? Thanks!

File: powerbi-visual-builder/src_extension/builder.ts [hunk 1]
```diff
@@ -234,8 +246,48 @@ class PowerBIVisualGenerator implements ExportTemplateTarget {
             type = { text: true };
           }
         }
+        if (typeof property.target.property === "object" &&
```
Comment (stopyoukid) [inline powerbi-visual-builder/src_extension/builder.ts:249]: I'd pull this out into a variable, `const targetProperty = property.target.property`, and replace all of the below instances of `property.target.property`, it makes it easier to read (for me at least)

File: powerbi-visual-builder/src_extension/builder.ts [hunk 2]
```diff
@@ -234,8 +246,48 @@ class PowerBIVisualGenerator implements ExportTemplateTarget {
...
+                displayName: enumeration[0].toLocaleUpperCase() + enumeration.slice(1),
+                value: enumeration
+              };
+            })
+          };
+        }
         break;
       }
+      case "object": {
+        if (property.target.property && (property.target.property as any).property === "sublayout") {
```
Comment (stopyoukid) [inline powerbi-visual-builder/src_extension/builder.ts:265]: Same comment as above

File: powerbi-visual-builder/src_visual/visual.ts [hunk 3]
```diff
@@ -547,12 +560,40 @@ namespace powerbi.extensibility.visual {
                 continue;
               }
               if (property.target.property) {
-                CharticulatorContainer.ChartTemplate.SetChartProperty(
-                  chart,
-                  property.objectID,
-                  property.target.property,
-                  this.properties[property.powerBIName]
-                );
+                if (typeof property.target.property === "object" &&
```
Comment (stopyoukid) [inline powerbi-visual-builder/src_visual/visual.ts:563]: Same variable comment here.

File: powerbi-visual-builder/src_visual/visual.ts [hunk 4]
```diff
@@ -740,11 +765,34 @@ namespace powerbi.extensibility.visual {
                 continue;
               }
               if (property.target.property) {
-                this.chartContainer.setProperty(
-                  property.objectID,
-                  property.target.property,
-                  this.properties[property.powerBIName]
-                );
+                if (typeof property.target.property === "object" &&
```
Comment (stopyoukid) [inline powerbi-visual-builder/src_visual/visual.ts:768]: Same variable comment

File: powerbi-visual-builder/src_visual/visual.ts [hunk 5]
```diff
@@ -245,7 +251,10 @@ namespace powerbi.extensibility.visual {
       }
 
       const tooltipsTable = this.getTooltipsTable(this.template);
-      const tooltipsTableColumns = options.dataViews[0].categorical.categories.filter(cat => cat.source.roles.powerBITooltips)
+      const tooltipsTableColumns = [
```
Comment (stopyoukid) [inline powerbi-visual-builder/src_visual/visual.ts:254]: I'd pull out a variable for `options.dataViews[0].categorical` here

**Pull request merged**